docs/producers/websocket.rst

```diff
@@ -97,3 +101,4 @@ Example
 	    Address: ":81"
 	    Path:    "/"
 	    ReadTimeoutSec: 3
+        IgnoreOrigin: false
```

Comment:
It looks that the indenting does not fit. Don't know (yet) if this will end in a different rendering result.